src/cbc_sdk/platform/vulnerability_assessment.py

```diff
@@ -104,6 +104,50 @@ def _query_implementation(cls, cb, **kwargs):
         """
         return VulnerabilityQuery(cls, cb)
 
+    def perform_action(self, type, reason=None, notes=None):
+        """
+        Take an action on to manage the Vulnerability.
```

Comment:
Think this should be one of:
Take an action on the Vulnerability.
Take an action to manage the Vulnerability.